src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.scss

```diff
@@ -92,6 +93,10 @@
   padding: 0;
 }
 
+:host ::ng-deep .mat-form-field-infix {
```

Comment:
I understand that this should basically apply to all form fields in the component, but just in case, add an additional class, something like `no-padding-form-field` or similar